src/asynqp/channel.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, id, synchroniser, sender, basic_return_consumer, queue_factor
         self.reader = reader
 
     @asyncio.coroutine
-    def declare_exchange(self, name, type, *, durable=True, auto_delete=False, internal=False):
+    def declare_exchange(self, name, type, *, durable=True, auto_delete=False, internal=False, arguments={}):
```

Comment:
The parameter is not modified in the children, so we should be fine.
What speaks against **kwargs, is that the arguments have quite often a hyphen in them, the arguments part is close to the specification of the protocol.